### PR TITLE
Newsletters: add #23 (2018-11-27)

### PR DESCRIPTION
**Pull request by harding**

### CODE REVIEW

File: _posts/en/newsletters/2018-11-27-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+- [C-Lightning #2096][] strips the text `lightning:` prefixed to a
+  [BOLT11][] invoice before attempting to process it.  This text is
+  sometimes added so that LN wallets can register for it as URI
+  handlers.  The prefix text will be striped if it is all lowercase or
+  all uppercase (but not mixed case) per the [BIP173][] bech32
+  specification.
+
+- C-Lightning [#2081][c-lightning #2081] and [#2092][c-lightning #2092]
+  fix a problem with running multiple RPC commands in parallel.  As a
+  user-visible change, lightningd now adds a double newline (`\n\n`)
```
Comment (practicalswift) [inline _posts/en/newsletters/2018-11-27-newsletter.md]: Should be `lightningd` instead fo lightningd to make it consistent with how `bitcoind` is written below? :-)

File: _posts/en/newsletters/2018-11-27-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments of time to help answer other people's questions.  In
+this monthly feature, we highlight some of the top voted questions and
+answers made since our last update.*
+
+- [How could you create a fake signature to pretend to be
+  Satoshi?][bse 81115] Gregory Maxwell asks and answers a question about
```
Comment (practicalswift) [inline _posts/en/newsletters/2018-11-27-newsletter.md]: Trailing space on this line :-)

File: _posts/en/newsletters/2018-11-27-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+%}
+
+- [Bitcoin Core #14708][] prints a warning when unrecognized section
+  names are used in the `bitcoin.conf` configuration file.  For example,
+  if you create the following configuration file using the name
+  `testnet` instead of the correct name `test`, Bitcoin Core would
+  previously silently ignore the testnet options.  This merged PR causes
+  it to print a notice: "Warning: Section [testnet] is not recognized."
+
+
```
Comment (practicalswift) [inline _posts/en/newsletters/2018-11-27-newsletter.md]: Nit: Multiple consecutive blank lines – not needed in Markdown :-)

File: _posts/en/newsletters/2018-11-27-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+  in hashrate during the next week, a difficulty adjustment is expected
+  around Sunday that will mitigate most of the recent hashrate reductions.
+
+## News
+
+- **Sighash updates:** Pieter Wuille started a [thread][wuille sighash]
+  on the Bitcoin-Dev mailing list suggesting two additions for future
+  changes to segwit sighashes, especially [BIP118][]
+  `SIGHASH_NOINPUT_UNSAFE`.  A signature hash is the data committed to
+  by a signature in a transaction. Normally the hash commits to a list
```
Comment (practicalswift) [inline _posts/en/newsletters/2018-11-27-newsletter.md]: Nit: Inconsistent spacing. Should be two spaces to make it consistent with rest of the text :-)

I don't how detailed you want the style nits. Please clarify if these nits are too detailed.

File: _posts/en/newsletters/2018-11-27-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+  of which coins are being spent, which scripts are receiving the coins,
+  and some metadata---but it's possible to sign only some of the
+  transaction fields in order to allow other users to change your
+  transactions in specific ways you might find acceptable (e.g. for
+  layer-two protocols).
+
+    Wuille suggests two additions to what metadata is hashed: first, the
+    transaction fee is included in the hash in order to ensure hardware
+    wallets or offline wallets can be absolutely sure that they aren't
```
Comment (practicalswift) [inline _posts/en/newsletters/2018-11-27-newsletter.md]: Style nit: Would changing from "absolutely sure" to "sure" change the meaning? :-)

**Comment by harding** [comment]
Pushed update (force push as GitHub now provides diffs for those) that should resolve nits.  Thanks for the review!

@practicalswift I'm happy to receive nits for extra whitespace at ends of lines and excessive newlines.  I think either one or two spaces after full stops are fine even with a single author (and allowing them makes accepting contributions from multiple authors much easier).

File: _posts/en/newsletters/2018-11-27-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+  changes to segwit sighashes, especially [BIP118][]
+  `SIGHASH_NOINPUT_UNSAFE`.  A signature hash is the data committed to
+  by a signature in a transaction. Normally the hash commits to a list
+  of which coins are being spent, which scripts are receiving the coins,
+  and some metadata---but it's possible to sign only some of the
+  transaction fields in order to allow other users to change your
+  transactions in specific ways you might find acceptable (e.g. for
+  layer-two protocols).
+
+    Wuille suggests two additions to what metadata is hashed: first, the
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/en/newsletters/2018-11-27-newsletter.md]: Thanks for changing this. I didn't include my rationale for wanting this to be more explicit. I wanted it to be clear that users wouldn't need to update all of their signing software if this change was introduced.

**Comment by jnewbery** [comment]
tested ACK 51825918befa097f9dec31a639a7df0bf1f486a6. Thanks!

**Comment by moneyball** [comment]
utACK. Made a minor typo fix.

**Comment by jnewbery** [comment]
Squashed @moneyball's commit and merged. Thanks all!

**Pull request merged**